2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -819,7 +833,112 @@ Le task sono organizzate in tre colonne, rappresentanti lo stato:
+- osservazione delle criticità delle soluzioni software preesistenti.
+Le comunicazioni con gli stakeholder avvengono nelle modalità descritte in @modalita-comunicazione.
+
+Le attività sono supportate, quando utile, da documenti, immagini, dimostratori tecnologici e in generale qualsiasi elemento informativo utile alla comprensione dei bisogni degli stakeholder. Le informazioni sono organizzate in modo da supportare l'identificazione, l'analisi e la trasformazione dei bisogni in requisiti. Il livello di astrazione adottato può differire in base all'interlocutre e al progresso globale conseguito dal _Processo di definizione di bisogni e requisiti degli stakeholder_.
+
+Lo strumento adottato a supporto di queste operazioni è Miro (TODO).
+
+I bisogni espressi da Committente e Fornitore sono raccolti, catalogati, analizzati ed espressi nel documento di _Analisi dei Requisiti_ prodotto dal gruppo.
+
+Alcuni stakeholder possono avere interessi avversi a quelli del gruppo o in contrasto con gli interessi di altri stakeholder. Qualora gli interessi degli stakeholder siano tra di essi contrastanti, ma non siano avversi al gruppo o al sistema software, il gruppo si adopera per mediare i contrasti. La strategia di mediazione prevede l'identificazione di un sottoinsieme di interessi e bisogni comuni, il confronto con le parti e la definzione di strategie di mediazione calate nella fattispecie.
+
+Gli intenti o i desideri di chi si oppone al gruppo o ad uno o più dei processi di ciclo di vita del sistema software (descritti in TODO) sono affrontati tramite il processo di gestione dei rischi TODO.
+
+Il negoziato tra le parti potrebbe essere richiesto per mediare posizioni mutualmente incompatibili, o a causa di vincoli o budget insufficiente. Anche la data di consegna prevista incide sulla realizzazione dei requisiti. Sarà sempre necessario consultare gli stakeholder coinvolti per raggiungere un accordo. Le decisioni saranno tracciate e rese disponibili agli stakeholder.
+
+==== Identificazione dei bisogni <identificazione-bisogni>
+Include l'elicitazione dei bisogni direttamente dagli stakeholder o dalla documentazione fornita da essi; oppure la raccolta di bisogni impliciti basati sul dominio applicativo ed i contesti tecnologico, legislativo, normativo.
+
+I bisogni degli stakeholder scaturiscono da fonti diverse. Il gruppo si impegna ad esplorare e valutare, al fine di identificare possibili bisogni, almeno questi frangenti:
+- Obiettivi di alto livello che il sistema dovrebbe conseguire;
+- Contributi concreti che il sistema dovrebbe apportare a beneficio degli stakeholder;
+- Scenari operativi, utili per limitare l'ambito e comprendere le aspettative e i bisogni;
+- Scenari operativi quotidiani, utili per assegnare una priorità ai bisogni;
+- Tipologie e caratteristiche degli utenti;
+- Ambiente operativo e contesto d'utilizzo;
+- Aspettative sulle prestazioni e la disponibilità del sistema;
```

Comment:
non credo che questo rientri nei nostri compiti, bassani ha detto più volte che non ha pretese sulle prestazioni (giusto che funzioni forse), nè credo sulla disponibilità del sistema.

Reply:
Infatti qui parliamo di aspetti che "il gruppo si impegna ad esplorare e valutare". Non ne sono stati individuati ma for lack of trying

Reply:
si, immaginavo sarebbe stata questa la risposta, ho comunque voluto evidenziare la cosa. top